### PR TITLE
Fix the size of the SCHEDULE EXAM button

### PR DESCRIPTION
**Pull request by annagav**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [ ] Mobile width screenshots
  - [ ] Tag @ferdi or @pdpinch for review
- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
Fix https://trello.com/c/a06fND6x/389-css-bug-schedule-an-exam-button-is-too-tall

#### What's this PR do?
Fix the size of the SCHEDULE EXAM button

#### How should this be manually tested?
Set up your dashboard, so that you enrolled in a course or you passed a course. You can use the `alter_data` management command to set that up.  Then create an available exam for scheduling:
```
from courses.models import *
from django.contrib.auth.models import User
from exams.factories import *
from exams.models import *
course = Course.objects.filter(title='Digital Learning 200')[0]

 exam_run = ExamRunFactory.create(course=course, authorized=True)
 user = User.objects.filter(username='student10')[0]

au=ExamAuthorizationFactory.create(user=user, course=course, exam_run=exam_run, status= 'success', exam_taken=True)
exam_profile= ExamProfileFactory.create(status='success', profile=user.profile)
```

Screen Shot 2020-08-12 at 4 20 27 PM

### CODE REVIEW

File: static/scss/_button.scss [hunk 1]
```diff
@@ -263,6 +263,7 @@
   font-weight: 400;
   line-height: 36px;
   border-radius: 2px;
+  height: fit-content;
```
Comment (asadiqbal08) [inline static/scss/_button.scss:266]: @annagav would you like to add a screen with more list items along the button ?

Comment (asadiqbal08) [reply static/scss/_button.scss:266]: Ok, I looked around the `fit-content` property and seems good to me in this scenario but would be nice if you add a screen with more list items along the button

**Comment by annagav** [comment]
Screen Shot 2020-08-25 at 10 11 57 AM

**Review by asadiqbal08 — APPROVED** [review]
👍

**Pull request merged**